cmake/compile-flags.cmake

```diff
@@ -17,7 +17,7 @@ list(APPEND CMAKE_EXE_LINKER_FLAGS
 list(JOIN CMAKE_EXE_LINKER_FLAGS " " CMAKE_EXE_LINKER_FLAGS)
 
 list(APPEND CMAKE_CXX_FLAGS
-        -std=gnu++20 -Wall -Werror -Qunused-arguments
+        -std=gnu++20 -Wall -Werror -Qunused-arguments -Wimplicit-fallthrough
```

Comment:
Sorry for the noise here. I just spent way too much time debugging problems that were result of missing break; This just helps me to remember that `switch` is not `match`.